File: SoftwaretechnikProjekt/Client/CommunicatioClient/ObjectTest/Hobby.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ObjectTest
{
    class Hobby
    {
        public Hobby() { }
        public Hobby(string hn, bool ish)
        {
            isSuperhobby = ish;
            hobbyname = hn;
        }
        public bool isSuperhobby = false;
        public string hobbyname = "";
    }
}

File: SoftwaretechnikProjekt/JavaJsonMQTest/src/entities/Hobby.java
package entities;


public class Hobby {
	public Hobby() {}
    public Hobby(String hn, boolean ish)
    {
        isSuperhobby = ish;
        hobbyname = hn;
    }
    public boolean isSuperhobby = false;
    public String hobbyname = "";
}
